resume-buddy/src/main/java/com/google/sps/servlets/MatchServlet.java

```diff
@@ -13,7 +13,8 @@
 /** Servlet that calls matching algorithm */
 @WebServlet("/match")
 public class MatchServlet extends HttpServlet {
-
+  
+  /** Called every 12 hours which activates matching algorithm */
```

Comment:
Nit: Since this servlet is not actually responsible for the 12 hour frequency, I think documenting that here doesn't really fit. Maybe document that this function must be called periodically to process matches?